### ISSUE TITLE
What does (31 to 16 => a_in(15)) mean?

### ISSUE THREAD

**Issue by dalxung** [edwar-vhd/SFU#5]
in convert_int_int.vhd
When "100" => a_tmp := (31 to 16 => a_in(15)) & a_in(15 to 0);
There is a syntax.
Are both the source and destination of (31 downto 16 => a_in(15)) a_in and assign the source bit string range from 31 to 16 to the 15th position of the target bit string?
Can you fit 16 bits into 1 bit space? Isn't that incorrect syntax?

**Comment by divadnauj-GB** [comment]
I do not think this question belongs to this repository, we do not have any design file called "convert_int_int.vhd" in this specific repository
